build.sh

```diff
@@ -38,8 +38,16 @@ function test_docker_container() {
   echo "Done checking links"
 }
 
+# Custom cleanup of CDN
+function clean_docker_container_cdn() {
+  clean_docker_container $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME
+
+  # Cleanup CDN
+  rm -rf cdn/deploy
```

Comment:
We probably don't need to clean this up -- it's only a cache so doesn't break things if it is present.